AAI/AAIConnectProfile.md

```diff
@@ -592,12 +652,12 @@ component "<b>Visa Issuer</b> (2)" as VisaIssuer2
 
 note "Signed visas can be sourced from\nmultiple visa issuers - either on\ndemand or via batch transfer/cached" as VisaNote
 
-VisaIssuer1 --> VisaNote : Visa A, B
-VisaIssuer2 --> VisaNote : Visa C
+VisaIssuer1 --> Broker  #text:red : (step 3) visas obtained
+VisaIssuer2 --> Broker 
 
-Client <-- Broker #text:red : (step 1) login flow results in\nan AAI passport-scoped access token
+Client <--> Broker #text:red : (step 1) login flow results in\nan AAI passport-scoped access token
 Client ---> TokenEndpoint #text:red : (step 2) request for token exchange
-VisaNote ---right---> PassportIssuer #text:red : (step 3) visas obtained
+Client ---> PassportIssuer #text:red : (step 3) visas obtained
 Client <-- TokenEndpoint  #text:red : (step 4) passport issued (passport contains visa A,B,C)
```

Comment:
The markings A,B,C were removed from step 3, so step 4 should not mention them.
```suggestion
Client <-- TokenEndpoint  #text:red : (step 4) passport issued (passport contains visas)
```

Reply:
Replaced the Passport Issuer Conformance diagram with a sequence diagram, which should address this feedback.